modules/likes/jetpack-likes-settings.php

```diff
@@ -578,6 +613,20 @@ function admin_settings_callback() {
 				delete_option( 'disabled_reblogs' );
 				break;
 		}
+
+		// WPCOM only: Comment Likes
+		if( ! $this->in_jetpack ) {
```

Comment:
Another super-nitpicky coding standard change (non-blocking, just to make code editors happy): missing space after `if`.